composer.json

```diff
@@ -10,7 +10,8 @@
         }
     ],
     "require-dev": {
-        "composer/composer": "1.0.*@dev"
+        "composer/composer": "1.0.*@dev",
+        "cakephp/core": "dev-master"
```

Comment:
It won't clone if you use `--prefer-dist` when installing / updating which I personally always do :)  Nonetheless it should be updated once we have a stable release.